optimizer/README.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-`amp-toolbox-ssr` is a library for server-side-rendering AMP pages. This makes it possible to optimize
+`amp-toolbox-optimizer` is a library for server-side-rendering AMP pages. This makes it possible to optimizer
 page loading times when serving AMP pages from a non-cache domain.
 
 **Warning:** server-side rendered AMP files are no longer valid AMPHTML. Hence, serve-side rendered AMP pages should link to their [valid AMP counterpart](https://www.ampproject.org/docs/guides/discovery).
```

Comment:
s/optimizer/optimize - In this case, optimize is not the project name, but part of the text.

Reply:
done